include/rive/assets/file_asset_contents.hpp

```diff
@@ -1,11 +1,18 @@
 #ifndef _RIVE_FILE_ASSET_CONTENTS_HPP_
 #define _RIVE_FILE_ASSET_CONTENTS_HPP_
 #include "rive/generated/assets/file_asset_contents_base.hpp"
+#include <cstdint>
 
 namespace rive {
     class FileAssetContents : public FileAssetContentsBase {
+    private:
+        std::vector<uint8_t> m_Bytes;
+
     public:
+        const std::vector<uint8_t>& bytes() const;
```

Comment:
Do any of the callers care that we are storing this in a std::vector? I wonder if its just as useful (and more general) to just return a span.

Span<const uint8_t> bytes() const { return {m_Bytes.data(), m_Bytes.size()}; }

Reply:
Good point, totally.